app/shop.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+

```

Comment:
There is no need to add a blank line here since both imports belong to the same group. Fix this issue in the `app/customer.py` as well